packages/x-scheduler/src/joy/event/agenda-event/AgendaEvent.css

```diff
@@ -0,0 +1,50 @@
+.EventContainer {
+  container-type: inline-size;
+  display: flex;
+  align-items: flex-start;
+  gap: var(--space-3);
+  flex-wrap: nowrap;
+}
+
+.EventCalendarResourceLegendColor {
+  margin: calc((var(--line-height-1) - 10px) / 2) 0;
+}
+
+.EventContent {
+  display: flex;
+  flex-wrap: wrap-reverse;
+  gap: var(--space-1);
+  flex-grow: 1;
+}
+
+@container (width < 300px) {
```

Comment:
https://caniuse.com/mdn-css_at-rules_container

Would be worth checking with the core team if we are fine using it (I think we support iOS Safari 15.8 which doesn't have this CSS rule, but it's the only I could find that would be problematic)

Reply:
Would it be ok if we left a comment here to double check this css rule before the mvp and confirming with the core team in the mean time? When we design/develop a responsive version, this might change

Reply:
Fine for me